### PR TITLE
✨ Add Makefile to the project

### PR DESCRIPTION
**Pull request by YosefAshenafi**
Features:

- Make file added

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+help:
+	@cat Makefile | grep '^## ' --color=never | cut -c4- | sed -e "`printf 's/ - /\t- /;'`" | column -s "`printf '\t'`" -t
+
+## install - install dependency packages
+install:
+	yarn install
+
+## dev - starts the Next.js development server on port 3000
+dev: install
+	yarn run dev
```
Comment (gwainor) [inline Makefile]: Convert to `npm` please. We are not using `yarn` on this one.
`npm run dev`
Apply rest of the file

**Review by gwainor — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (YosefAshenafi) [reply Makefile]: done

**Review by jontsai — APPROVED** [review]
LGTM

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+run: install
+	npm run build && npm run start
+
+## clean - clean previous builds
+clean:
+	rm -rf out/*
+
+## build - build the app for release
+build: clean install
+	npm build
```
Comment (gwainor) [inline Makefile]:
```suggestion
	npm run build
```

**Review by gwainor — APPROVED** [review]
LGTM with a small comment

File: Makefile (hunk 2 above)
Comment (YosefAshenafi) [reply Makefile]: done

**Pull request merged**